.github/workflows/ci.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x, 17.x]
+        node-version: [22.x, 24.x, 25.x]
```

Comment:
woah that's quite the bump!

Looks like we should include `20` until April 2026.
Source: https://endoflife.date/nodejs
<img width="730" height="803" alt="image" src="https://github.com/user-attachments/assets/27028289-b277-4dfe-90d3-e90f8f5a6254" />